src/test/java/com/cdancy/bitbucket/rest/features/RepositoryApiMockTest.java

```diff
@@ -106,7 +106,6 @@ public void testGetRepositoryNonExistent() throws Exception {
         try {
             String projectKey = "PRJ";
             String repoKey = "notexist";
-            CreateRepository createRepository = CreateRepository.create(repoKey, true);
```

Comment:
Again, while cleanup and code fixing is great, lets create a separate PR for this work and remove commit `b301b4888e920866065c17a5f30e6583a37d5a52` as it doesn't pertain to `PullRequestCommentsApi`.